cobalt/browser/BUILD.gn

```diff
@@ -83,3 +85,21 @@ action("embed_polyfilled_javascript") {
     rebase_path("embedded_resources", root_build_dir),
   ]
 }
+
+source_set("metrics") {
+  # TODO(cobalt, b/375655377): remove testonly declaration, needed because of
+  # being depended-on by :browser above.
+  testonly = true
```

Comment:
Just curious, what are we doing in the interim w/ testonly? Do we have to virally propagate this to _all_ targets in the binary?

Reply:
Yes, sadly the `testonly` spreads to all targets depending on content_shell. It's very unfortunate and it's a design decision trading off relying on `content_shell` for expediency.